content/blog/2010/2010-04-24-hudson-1-355-released.md

```diff
@@ -32,16 +32,13 @@ I won't go into details on some of the infrastructure changes we have lined up j
   <li class=bug> 
     Fixed a bug where a job created via XML didn't properly receive upstream/downstream computation.
     (<a href="https://n4.nabble.com/Hudson-API-td1747758.html#a1747758">report</a>)
-  <li class=bug> 
```

Comment:
The nabble link does not work, so it can be deleted.  The reference to the bug fix seems like it should be retained and have the word 'slave' changed to 'agent'